### ISSUE TITLE
[Bug]: Could not find Chromium

### ISSUE THREAD

**Issue by LMS5413** [puppeteer/puppeteer#9581]
### Bug expectation

Using [Gigalixir Host](https://gigalixir.com/)
Installing buildpack from heroku.
Start the application

### Bug behavior

- [ ] Flaky
- [ ] PDF

### Minimal, reproducible example

```TypeScript
const puppeteer = require('puppeteer');
const browser = await puppeteer.launch({headless: true, args: ['--no-sandbox', '--disable-setuid-sandbox']});
```

### Error string

 Could not find Chromium (rev. 1083080). This can occur if either\n 1. you did not perform an installation before running the script (e.g. `npm install`) or\n 2. your cache path is incorrectly configured (which is: /app/.cache/puppeteer).\nFor (2), check out our guide on configuring puppeteer at https://pptr.dev/guides/configuration. ![image](https://user-images.githubusercontent.com/58144443/214613317-c57e2a8a-920a-4e6b-b88b-de265f991808.png)

### Puppeteer version

19.6.0

### Node version

19.4.0

### Package manager

yarn

### Package manager version

1.22.18

### Operating system

Linux

**Comment by OrKoN** [comment]
Could you verify that Heroku downloads the browser to /app/.cache/puppeteer? Also try changing the cache directory to be in the local folder https://pptr.dev/guides/configuration#changing-the-default-cache-directory

**Comment by VictorAssis** [comment]
Thanks @OrKoN ! I solved the problem changing the cache directory to local folder and using the `jontewks/puppeteer` buildpack.

For anyone experiencing the same issue, I created the puppeteer config file (`puppeteer.config.cjs`) in the root of the project to change the cache location:
```
const { join } = require('path');

/**
 * @type {import("puppeteer").Configuration}
 */
module.exports = {
  // Changes the cache location for Puppeteer.
  cacheDirectory: join(__dirname, '.cache', 'puppeteer'),
};
```
(I recommend putting the `.cache` folder in `.gitignore` to not send it to the repository.)

After that, install and uninstall `puppeteer`, deploy it and check if the browser is located in the `/app/.cache/puppeteer` folder using the command `heroku run bash -a your-app-name` to access heroku server.

If it is there, use the command below to add the buildstack that will install all the dependencies for the browser to work correctly:
```
heroku buildpacks:add jontewks/puppeteer -a your-app-name
```

Re-deploy and it should be working!
